README.md

```diff
@@ -43,8 +43,14 @@ This quickstart uses Entando's standard Wildfly image.
    most of these settings can be set directly on the Entando custom resource after deployment too.
 
 4. Process the template and deploy the output using your favorite Kubernetes client, e.g:
-   
-      `helm template --name=quickstart  ./ | kubectl apply -n [your-sandbox-namespace] -f -`
```

Comment:
@ampie please looks like helm2 (version tested: v2.17.0) assumes the default namespace if namespace parameter is not specified.